File: backend/check_schema.js
const { masterPool } = require('./utils/db');

async function checkSchema() {
    try {
        const res = await masterPool.query(`
            SELECT column_name, data_type 
            FROM information_schema.columns 
            WHERE table_name = 'usuarios'
        `);
        console.log('--- Colunas da tabela usuarios ---');
        console.table(res.rows);
        process.exit(0);
    } catch (err) {
        console.error(err);
        process.exit(1);
    }
}
checkSchema();

File: bi-engine/check_schema.py
import sys
sys.path.insert(0, '.')

from sqlalchemy import create_engine, text
from config import DATABASE_URL

engine = create_engine(DATABASE_URL)

with engine.connect() as conn:
    # Check pedidos columns
    result = conn.execute(text("""
        SELECT column_name, data_type
        FROM information_schema.columns
        WHERE table_name = 'clientes'
        ORDER BY ordinal_position
    """))
    
    print("=== COLUNAS DA TABELA CLIENTES ===")
    for row in result.fetchall():
        print(f"  {row[0]}: {row[1]}")
